app/templates/play/level/tome/spell_palette_entry_popover.jade

```diff
@@ -1,7 +1,9 @@
 h4
   span.prop-name= doc.shortName
   |  - 
-  code.prop-type= doc.type == 'function' && doc.owner == 'this' ? 'method' : doc.type
+  code.prop-type(data-i18n='skill_docs.'+(doc.type == 'function' && doc.owner == 'this' ? 'method' : doc.type))
+    // In case there's no translation in en.coffee because we missed one
+    = (doc.type == 'function' && doc.owner == 'this') ? 'method' : doc.type
```

Comment:
Slightly cleaner to create a local var for this:

```jade
  - var skillType = doc.type == 'function' && doc.owner == 'this' ? 'method' : doc.type;
  code.prop-type(data-i18n='skill_docs.' + skillType)
    // In case there's no translation in en.coffee because we missed one
    = skillType
```